vignettes/geologic-map.Rmd

```diff
@@ -39,46 +39,46 @@ install.packages("devtools")
 
 ```{r def_strat_names, eval = TRUE}
-# Get information for the chosen formation
-hc_def <- def_strat_names(strat_name = "hell creek")
-hc_def
+# Search stratigraphic names for Hell Creek
+def_strat_names(strat_name = "hell creek")
 ```
-So we can see here that there are three different `strat_name`s associated with the Hell Creek. You'll also notice that these all contain the same `concept_id`, showing that this is one group of `strat_name`s which hopefully represent the same geologic entity. We can therefore use this `concept_id` to ensure we've got all mentions of the Hell Creek:
+We can see that there are three different `strat_name`s associated with the name Hell Creek. You'll also see that all three have the same `concept_id`, showing that these stratigraphic names are united under a single geologic entity. We can therefore use this `concept_id` to search for all references to the Hell Creek:
 
 ```{r check concept, eval = TRUE}
-# Get information for the chosen formation
+# Search stratigraphic names for all associated with Hell Creek concept_id
 def_strat_names(concept_id = 8598)
 ```
-Yup, that's all the entries for the Hell Creek. But hang on! Before going ahead and using these to pull our spatial data, you'll also notice that one of the `strat_name`s here is actually the Hell Creek Member, which belongs to the Lance formation. Let's try getting the information again, this time specifying the stratigraphic rank of "Formation":
+Yup, these three are the only entries for Hell Creek. But hang on! Before going ahead and using these to pull our spatial data, you'll also notice that one of the `strat_name`s here is actually the Hell Creek "Member", which is described as belonging to the Lance Formation. Let's try getting the information again, this time specifying the stratigraphic rank of "Formation":
 
 ```{r strat_name with rank, eval = T}
-# Get information for the chosen formation, specifying by stratigraphic rank
+# Search stratigraphic names for formations called Hell Creek
 hc_def <- def_strat_names(strat_name = "hell creek", rank = "Fm")
+# View data
 hc_def
 ```
-That's much better. Let's go ahead and use these `strat_name_id`s to get some spatial data. To do this, we'll need to use the `get_map_outcrop()` function, which allows us to grab outcrop polygons from geological maops. Let's do that now, specifying we want the output to be returned as an sf object. 
+That's much better. Let's go ahead and use these `strat_name_id`s to get some spatial data. To do this, we'll need to use the `get_map_outcrop()` function, which allows us to grab outcrop polygons from geological maps. Let's do that now, specifying that we want the output to be returned as an sf object.
```

Comment:
Many readers won't be familiar with `sf` objects. Can we add a brief description of what this means, and/or why this is the best format for the data?

Reply:
good shout, and sure thing :)